packages/theme/src/colors.ts

```diff
@@ -23,7 +23,26 @@ const backgroundElevationColors = {
     },
 } as const;
 
+const backgroundTertiaryElevationColors = {
+    standard: {
+        backgroundTertiaryElevationNegative: '#f6f6f6ff', // @TODO
+        backgroundTertiaryDefaultOnElevation0: '#eeeeeeff',
+        backgroundTertiaryDefaultOnElevation1: '#f6f6f6ff',
+        backgroundTertiaryDefaultOnElevation2: '#eeeeeeff', // @TODO
+        backgroundTertiaryDefaultOnElevation3: '#f6f6f6ff', // @TODO
+    },
+    dark: {
+        backgroundTertiaryElevationNegative: '#1c1e1cff', // @TODO
+        backgroundTertiaryDefaultOnElevation0: '#161716ff',
+        backgroundTertiaryDefaultOnElevation1: '#1c1e1cff',
+        backgroundTertiaryDefaultOnElevation2: '#161716ff', // @TODO
+        backgroundTertiaryDefaultOnElevation3: '#1c1e1cff', // @TODO
+    },
+} as const;
```

Comment:
How about separating by domains? Text, Button, Icon, etc?

Reply:
I will create an issue for it